### PR TITLE
DevEx: Remove Document.caseId 

### PR DESCRIPTION
**Pull request by ericsorenson**
Document.caseId appears to not be needed.

### CODE REVIEW

File: web-client/src/presenter/computeds/formattedCaseDetail.js [hunk 1]
```diff
@@ -197,7 +197,7 @@ export const formattedCaseDetail = (get, applicationContext) => {
         ...draftDocument,
         descriptionDisplay: draftDocument.documentTitle,
         editLink: '',
-        showDocumentEditLink: draftDocument && permissions.UPDATE_CASE,
```
Comment (rachelschneiderman) [inline web-client/src/presenter/computeds/formattedCaseDetail.js:200]: Out of curiosity, why was this removed?

**Review by rachelschneiderman — APPROVED** [review]

File: web-client/src/presenter/computeds/formattedCaseDetail.js (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/presenter/computeds/formattedCaseDetail.js:200]: Weird, I think this was a glitch in the GitHub UI. The line wasn't removed, just the check for `draftDocument`, which is already used earlier in the function. If it were falsy, the function would already have thrown an exception.
